### PR TITLE
[MM-55104] Replace usage of LocalizedIcon in 'rhs_header_post.tsx' with i/span tags

### PR DESCRIPTION
**Pull request by Sudhanva-Nadiger**
#### Summary
- fixes: #25106
- eliminate the usage of LocalizedIcon in the `webapp/channels/src/components/rhs_header_post/rhs_header_post.tsx`

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-55104

#### Screenshots

|before  |  after  |
|----|----|
| image |image |

|before  |  after  |
|----|----|
| image |image |

#### Release Note

```release-note
None
```

### CODE REVIEW

File: webapp/channels/src/components/rhs_header_post/rhs_header_post.tsx [hunk 1]
```diff
@@ -45,7 +43,7 @@ interface RhsHeaderPostProps {
     setThreadFollow: (userId: string, teamId: string, threadId: string, newState: boolean) => void;
 }
 
-export default class RhsHeaderPost extends React.PureComponent<RhsHeaderPostProps> {
+class RhsHeaderPost extends React.PureComponent<RhsHeaderPostProps> {
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/rhs_header_post/rhs_header_post.tsx:46]: nit lets rename `RhsHeaderPostProps` to Props

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
Can you please check if the test file for this component would require any changes?

**Comment by Sudhanva-Nadiger** [comment]
All tests are passing 🎉 

image
image

**Review by M-ZubairAhmed — APPROVED** [review]
Thanks for your remarkable contributions; your work is outstanding. 1️⃣ 0️⃣ 0️⃣ %

**Review by yasserfaraazkhan — APPROVED** [review]
verified in pinned, saved, threads, playbook.
works as expected.

**Comment by M-ZubairAhmed** [comment]
@jespino gentle ping on this one

**Review by jespino — APPROVED** [review]
Looks good to me. Thanks @Sudhanva-Nadiger! :confetti_ball:

**Comment by jespino** [comment]
@M-ZubairAhmed is this ready to merge?

**Pull request merged**